### PR TITLE
Improvements on importer and small bug fixes

### PR DESCRIPTION
**Pull request by ArneTR**
- can import tags and can handle empty tags
- can handle empty title
- correct created_at and updated_at
- close the root note

### CODE REVIEW

**Comment by tanrax** [comment]
Good job @ArneTR

**Comment by tanrax** [comment]
Could you remove the annotated code fragments?

**Comment by ArneTR** [comment]
I thougt it would make sense to keep them in.

I originally exported from Google Keep To Standard Notes and got these
escapes.

Also the debugging may help for some ... if you don't want them in your
code feel free to cut them out

Best,

Arne

**Comment by tanrax** [comment]
@ArneTR It is not a good practice to leave commented code as it is not self-explanatory. For this, perhaps, it would be advisable to take it to README.md with some commentary. I would appreciate it if you would apply either of these actions :)

File: standard-notes-to-enex.py [hunk 1]
```diff
@@ -21,14 +23,62 @@ def text_from_html(title, text):
...
+            else:
+                tag_notes_links[reference['uuid']] = [html.escape(item['content']['title'])]
+
+
+#print (json.dumps(tag_notes_links, indent=2))
+#quit()
+
+counter = 0
+for key, item in enumerate(data['items']):
+ if item['content_type'] == 'Note':
```
Comment (tanrax) [inline standard-notes-to-enex.py:44]: There's bad indentation.

Comment (ArneTR) [reply standard-notes-to-enex.py:44]: unclear what you mean by bad indentation ....? Maybe your editor got it wrong? Looks fine to me ...?

Anyhow, I closed the request and send you an new one.

Please have a look at it.

Also you can accept the pull-request and modify it afterwards if you want to have the code differently in your repo. Just wanted you to get the credit and keep it centralised

Comment (ArneTR) [reply standard-notes-to-enex.py:44]: sry, my bad. Saw it was just one space instead of usually 4. fixed

**Pull request closed without merging**